packages/search-ui/src/Filter/RatingFilter.tsx

```diff
@@ -5,21 +5,21 @@ import { useSearchUIContext } from '../ContextProvider';
 import ListFilter from './ListFilter';
 import { RatingFilterProps } from './types';
 
-const RatingFilter = ({ name, title, hideCount }: Omit<RatingFilterProps, 'type'>) => {
+const RatingFilter = ({ name, title, hideCount, ratingMax: overrideRatingMax }: Omit<RatingFilterProps, 'type'>) => {
   const { ratingMax, disableDefaultStyles, customClassNames } = useSearchUIContext();
```

Comment:
Just my personal opinion to keep the name of the component prop.

```suggestion
const RatingFilter = ({ name, title, hideCount, ratingMax }: Omit<RatingFilterProps, 'type'>) => {
  const { ratingMax: contextRatingMax, disableDefaultStyles, customClassNames } = useSearchUIContext();
```